frontend/dispatcher.lua

```diff
@@ -1231,4 +1231,28 @@ function Dispatcher:execute(settings, exec_props)
     end
 end
 
+function Dispatcher.updateActionNameInPlugins(ui, old_name, new_name)
```

Comment:
Might it not be simpler to not add anything in dispatcher and have a event handler `onUpdateActionName` in gestures, profiles etc and Broadcast it from footer, shortcuts etc?

Reply:
Agree, thanks.